api/.env

```diff
@@ -14,9 +14,11 @@ PGPASSWORD=www-data
 PGHOST=db
 PGPORT=5432
 PGDATABASE=swissgeol-local
+PG_SSL_MODE=disable
+# PG_SSL_MODE=require
```

Comment:
Will this stay here or is it a note until deployment?

Reply:
I left it there so we know what value we need if we want to enforce SSL.